openshift/install.sh

```diff
@@ -51,20 +51,20 @@ source $OPENSHIFT_HOMEDIR/python/virtenv/bin/activate
 
 cd ${OPENSHIFT_REPO_DIR}
 
-sh "python ${OPENSHIFT_REPO_DIR}/setup_weblate.py develop"
-
 # Pin Django version to 1.7 to avoid surprises when 1.8 comes out.
```

Comment:
Updated to Django 1.8. I'm pinning the Django version in the OpenShift configuration so that stable Weblate releases can still be installed on OpenShift when a newer, incompatible Django version is released.